cpu/esp32/bootloader/sdkconfig_default_esp32s2.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2022 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_esp32
+ * @{
+ *
+ * @file
+ * @brief       Default SDK configuration for the ESP32-S3 SoC bootloader
+ *
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ */
+
+#ifndef SDKCONFIG_DEFAULT_ESP32S2_H
+#define SDKCONFIG_DEFAULT_ESP32S2_H
+
+#ifndef DOXYGEN
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef CONFIG_ESP32S3_DEFAULT_CPU_FREQ_MHZ
+#define CONFIG_ESP32S3_DEFAULT_CPU_FREQ_MHZ 160
+#endif
+
+#define CONFIG_ESP32S3_DEBUG_OCDAWARE 1
```

Comment:
ESP32S2?

Reply:
> ESP32S2?

Good catch, a typical copy & paste error :worried: I fixed and squashed it directly.